### ISSUE TITLE
Error on streaming null values to CSV.Sink

### ISSUE THREAD

**Issue by nkottary** [JuliaDB/ODBC.jl#120]
This issue is in the latest checkout of `jq/remodel` branch.

I am streaming data from an `ODBC.Source` to a `CSV.Sink`.  This works when the source has no null values.  When I inserted a row with a null value for one of the columns, I got the following error:

```julia
julia> src = ODBC.Source(hndl, "select * from remodeltest.Employee;")
julia> sink = CSV.Sink(TEMP_FILE)
julia> Data.stream!(src, sink)
ERROR: type Sink has no field null
 in stream! at /home/Nishanth/.julia/v0.4/ODBC/src/backend.jl:135
```

`Data.stream!(src, Data.Table)` works properly.

**Comment by quinnj** [comment]
Will be fixed by 9792482faed2c3c3df9b95f7a7b985e5354d40b3

**Comment by nkottary** [comment]
:+1:
